k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -1166,13 +1213,16 @@ function deployCortxClient()
             --set cortxclient.sslcfgmap.name="cortx-ssl-cert-cfgmap-$namespace" \
             --set cortxclient.sslcfgmap.volmountname="ssl-config001" \
             --set cortxclient.sslcfgmap.mountpath="/etc/cortx/solution/ssl" \
+            --set cortxclient.machineid.value="$cortxclient_machineid" \
             --set cortxclient.localpathpvc.name="cortx-client-fs-local-pvc-$node_name" \
             --set cortxclient.localpathpvc.mountpath="$local_storage" \
             --set cortxclient.localpathpvc.requeststoragesize="1Gi" \
             --set namespace=$namespace \
             -n $namespace
     done
 
+exit 1
```

Comment:
Leftover debug code?

Reply:
D'oh! Good catch. I could have sworn I removed that and it wasn't causing problems in my testing.